realis.go

```diff
@@ -286,6 +287,21 @@ func (r realisClient) AddInstances(instKey aurora.InstanceKey, count int32) (*au
 	return response.ResponseCodeCheck(resp)
 }
 
+func (r realisClient) GetTaskStatus(query *aurora.TaskQuery)(tasks []*aurora.ScheduledTask, e error) {
+
+	resp, err := r.client.GetTasksStatus(query)
+	if err != nil {
+		return nil, errors.Wrap(err, "Error querying Aurora Scheduler for task status")
+	}
+	//Check for response code..
+	if resp.GetResponseCode() != aurora.ResponseCode_OK {
+		return nil, errors.New(resp.ResponseCode.String() + "--" + response.CombineMessage(resp))
```

Comment:
Return doesn't return aurora.Response as I previously thought which makes it right to return nil in this case instead of using response.ResponseCodeCheck().